src/main/java/build/buildfarm/worker/Worker.java

```diff
@@ -473,6 +473,7 @@ public static void main(String[] args) throws Exception {
     Path configPath = Paths.get(args[0]);
     InputStream configInputStream = Files.newInputStream(configPath);
```

Comment:
Prefer try-with-resources here to guarantee closing even if there's an exception.